src/plonemeeting/portal/core/browser/manage_users.py

```
@@ -284,3 +285,95 @@ def handleInvite(self, action):
+        # Pre-fill user info, almost all will be readonly as keycloak is managing those
+        # We only manage permissions
+        self.widgets["email"].value = member.getProperty("email", "")
+        self.widgets["email"].readonly = "readonly"
+        self.widgets["fullname"].value = member.getProperty("fullname", "")
+        self.widgets["fullname"].readonly = "readonly"
+        self.widgets["user_groups"].value = self.get_manageable_groups_for_user(username)
+
+    @button.buttonAndHandler(_plone("save"), name="save")
+    def handleSave(self, action):
+        """Create or update user, then update their group membership."""
+        data, errors = self.extractData()
```

Ruff (0.15.14)

[warning] Unpacked variable `errors` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)